.gitlab-ci.yml

```diff
@@ -407,6 +411,21 @@ pre_release_image:
     - export VERSION=`cat version.txt`
     - ./omnibus/package-scripts/publish_image.sh $CI_COMMIT_REF_NAME $STS_DOCKER_TEST_REPO $CI_PROJECT_DIR/outcomes/pkg
 
+pre_release_trace_agent:
+  stage: pre_release
+  image: docker:18.06.0
+  dependencies: 
+    - deps_deb
+    - build_deb
+  variables:
+    DOCKER_DRIVER: overlay
+  services:
+    - docker:dind
+  script:
+    - cd $CI_PROJECT_DIR
+    - export VERSION=`cat version.txt`
+    - ./omnibus/package-scripts/publish_trace_agent.sh $CI_COMMIT_REF_NAME $STS_DOCKER_TEST_REPO $CI_PROJECT_DIR/outcomes/binary/trace-agent
```

Comment:
don't use `$STS_DOCKER_TEST_REPO` instead define a variable at the top like the other `STS_DOCKER_TEST_REPO_TRACE: stackstate-trace-agent-test` and use it here

Reply:
added the test repo